Integrations/integration-Cisco-umbrella.yml

```diff
@@ -31,495 +31,1834 @@ configuration:
+                'Name': domain
+            }
 
-    var domainArg = (args.domain) ? args.domain : '';
-    domainArg = extractDomainName(domainArg);
+            contents = {
+                # Will be override in case result contains any
+                'Content Categories': 'No Content Categories Were Found',
+                'Malware Categories': 'No Security Categories Were Found'
+            }
 
-    // "one_two" to "One Two"
-    var keyToTableKey = function(s) {
-        return s && s.replace(/_/g, ' ')
-                .replace(/\w+/g, function(w){return w[0].toUpperCase() + w.slice(1).toLowerCase();});
-    }
+            if categorization:
+                if categorization.get('status'):
+                    contents['Status'] = categorization['status']
+            if categorization.get('content_categories'):
+                content_categories = ",".join(categorization['content_categories'])
+                contents['Content Categories'] = content_categories
+                domain_context['ContentCategories'] = content_categories
+            if categorization.get('security_categories'):
+                security_categories = ",".join(categorization['security_categories'])
+                contents['Malware Categories'] = security_categories
+                domain_context['SecurityCategories'] = security_categories
+            if categorization['status'] == -1:
+                domain_context['Malicious'] = {
+                    'Vendor': 'Cisco Umbrella',
+                    'Description': security_categories
+                }
 
-    // "one_two" to "OneTwo"
-    var keyToContextKey = function(s) {
-        return s && keyToTableKey(s).replace(/\s+/g, '');
-    }
+            context[outputPaths['domain']] = domain_context
+
+        # Build Output
+        results.append ({
+            # type
+            'Type': entryTypes['note'],
+            # contents
+            'ContentsFormat': formats['json'],
+            'Contents': contents,
+            # human-readable
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('Categorization:', contents, headers),
+            # context
+            'EntryContext': context
+        })
+        # Send table/context output to war-room
+        return results
+
+
+    def get_domain_categorization(domain):
+        # Build & Send request
+        endpoint_url = '/domains/categorization/' + domain + '?showLabels'
+        res = http_request(endpoint_url)
+
+        # Validate and assign response
+        categorization = res.get(domain, [])
+        if not categorization:
+            return False
+        return categorization
+
+
+    def get_domain_search_command():
+        # Initialize
+        contents = []
+        context = {}
+        headers = []
+        results = []
+        # Get vars
+        regex = demisto.args()['regex']
+        start = demisto.args().get('start','')
+        # Fetch data
```

Comment:
You can use blank lines to separate different code segments, that would be more readable this way